pkg/ecdsa/params/params_box_test.go

```diff
@@ -0,0 +1,69 @@
+package params
```

Comment:
Does it have to be a separate package? Can't we hold it in `tss` package?

Reply:
I don't want to hold it in `tss` because then the members of `tss` can refer to the box content directly while I want them to go through `Content` function.

Reply:
So maybe we could keep it under `/pkg/ecdsa/tss/params`? This package is strictly related to tss protocol, so we should probably keep it there.

Reply:
Good point. Moved it there; see https://github.com/keep-network/keep-ecdsa/pull/357/commits/159264584c822e5f3aa3bf9f61eebbb7ccc3f59f